### DISCUSSION TITLE
Edit climate schedule

### DISCUSSION THREAD

**Discussion by SukramJ** [SukramJ/aiohomematic#1761] [Q&A]
Gute Idee oder nicht?
Ab Version [1.67.0b2](https://github.com/danielperna84/custom_homematic/releases/tag/1.67.0b2) kann der Zeitplan eines Thermostates per HA-Action geändert werden.
Entsprechende [Details](https://github.com/danielperna84/custom_homematic#homematicip_localset_schedule_profile_weekday-experimental) finden sich in der README.
Die Regeln hab ich mit natürlich nicht ausgedacht, sondern sind aus den Daten oder aus dem Verhalten meiner Thermostate abgeleitet.
Eine UI Unterstützung kann ich dazu nicht bieten, aber wer Lust hat kann gerne aktiv werden.

**Comment by SukramJ** [comment]
@Baxxy13 Vielleicht interessiert dich das Thema, wenn ja, dann hätte ich gerne dein Feedback.

**Reply by Baxxy13** [reply]
Ich muss ja sagen das ich nicht so der "Wochenprogramm-Umschreiber" bin.
Weiß gar nicht wann ich die WP's das letzte Mal angefasst habe.

Aber grundsätzlich hast du das in der Readme gut erklärt und zumindest für mich ist alles verständlich.
Funktional ist es auch, also Daumen hoch von mir. 
(hab es zum Test auf ein WTH-B-2 und auch auf ein HmIP-Heizgruppen Gerät angewandt)

Edit: 
Du hast noch nen typo im letzten Abschnitt:
`retrieved a schedule with homematicip_local.set_schedule_profile_weekday`
muss doch...
`retrieved a schedule with homematicip_local.get_schedule_profile_weekday`
sein. 
Also `get` statt `set`.
